CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ option(FTL_VALGRIND "Link and test with Valgrind" OFF)
 option(FTL_FIBER_STACK_GUARD_PAGES "Add guard pages around the fiber stacks" OFF)
 option(FTL_CPP_17 "Enable C++17 features" OFF)
 option(FTL_WERROR "Promote compiler warnings to errors." OFF)
+option(FTL_DISABLE_ITERATOR_DEBUG "In MSVC, sets _ITERATOR_DEBUG_LEVEL=0. NOP for all other compilers." OFF)
```

Comment:
When you use this normally, do you actually not want any debugging? I remember you using at least level 1 in debug.

Reply:
If the user leaves the default option, then IMO, FTL shouldn't change the value at all. If they opt in, we'll change the value to match